rllib/offline/estimators/importance_sampling.py

```diff
@@ -16,37 +15,39 @@ class ImportanceSampling(OffPolicyEstimator):
     https://arxiv.org/pdf/1911.06854.pdf"""
```

Comment:
**nits**: It would be great if for documentation you could provide a TLDR gist of what is happening in this estimator using math docstring. Keep it as high level as you can so that the user can see the differences without reading the actual paper. For compiling math operation in markdown you can use a markdown compiler online and confirm that your math will render in the docs later.  For example, in importance sampling:
```
value_estimate = E_{tau ~{D}}(sum_{t=0}^{T} [\pi_{\text{target}}(a_t|s_t) / \pi_{\text{behavior}}(a_t|s_t)](r_t))
```

Reply:
Do we have an example of this somewhere else in the docs? I'm not sure how our docs handle math in markdown.

Reply:
I used some math operations in the CRR config. I did not compile them to be honest but the gist of the math is there at least.

Reply:
Done